digits/inference/job.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+from . import tasks
+import digits.frameworks
+from digits.job import Job
+from digits.utils import override
+
+class InferenceJob(Job):
+    """
+    A Job that exercises the forward pass of a neural network
+    """
+
+    @override
```

Comment:
Similarly, `@override` is a no-op on classes without `@subclass`. And you can always override `__init__` - there's no need to check that the parent class has that function.